src/patternfly/components/Popover/popover-title.hbs

```diff
@@ -1,9 +1,6 @@
-<div class="pf-c-popover__title{{#if popover-title--IsIcon}} pf-m-icon{{/if}}{{#if popover-title--modifier}} {{popover-title--modifier}}{{/if}}"
+<div class="pf-c-popover__title{{#if popover-title--modifier}} {{popover-title--modifier}}{{/if}}"
   {{#if popover-title--attribute}}
     {{{popover-title--attribute}}}
   {{/if}}>
-  {{#if popover-title--IsIcon}}
-    {{> popover-title-icon}}
-  {{/if}}
```

Comment:
Looks like no icon is being rendered anymore in the preview.  Instead of removing lines 5-7, could we instead update it to something like:

```suggestion
{{#if popover-title-icon--type}}
  {{> popover-title-icon}}
{{/if}}
```